doc/source/tutorials.rst

```diff
@@ -32,6 +32,7 @@ Helper Scripts
 - :ref:`dispersion.py <disp-script>`
 - :ref:`dos.py <dos-script>`
 - :ref:`optimise_eta.py <eta-script>`
```

Comment:
These should also be updated to `euphonic-dispersion` etc.

Reply:
done